doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
+A backend allows a file to be interpreted as keys in the key database such that any edits to the keys are reflected in the file and vice versa.  
+Additionally, the user can use this command to list the currently mounted backends by running the command with no arguments.  
+
+## USAGE
+
+`kdb mount [<path> <mountpoint>] [<plugin> [<config>] [..]]`  
```

Comment:
Yes, you could use it like that. In your example the `config=setting` would apply to both`plugin1` and `plugin2`. I cannot give you a good real-world example, because currently there are hardly plugins using the same configuration-keys.

Maybe, you can think of `-c verbose=1`, then all plugins supporting it would be verbose.